webapp/utils/aws/cloudwatch.py

```diff
@@ -27,7 +36,13 @@ class CloudWatchLogsClient:
     def client(self) -> "CloudWatchLogsClientType":
         return boto3.client("logs")
 
-    def get_logs(self, task_id: str) -> list[str] | None:
+    def get_log_messages(self, task_id: str) -> list:
+        messages: list = []
+        if logs := self.get_log_events(task_id):
+            messages.extend(event["message"] for event in logs)
+        return messages
```

Comment:
Could this `yield` or does it need to be a list?

Reply:
It needs to be a list. You can view this [sample function call in the code under development](https://github.com/MITLibraries/alma-sapinvoices-ui/blob/frontend-dev/webapp/app.py#L47) to see how `CloudWatchLogsClient` is used by the Flask app.